crates/common/mqtt_channel/src/connection.rs

```diff
@@ -200,8 +207,25 @@ impl Connection {
         mut event_loop: EventLoop,
         mut message_sender: mpsc::UnboundedSender<MqttMessage>,
         mut error_sender: mpsc::UnboundedSender<MqttError>,
+        awaiting_acks: mpsc::UnboundedReceiver<()>,
```

Comment:
I should replace this with something more appropriate now I'm no longer sending messages to it, I'm just using it to determine when the sender loop has finished executing.